meta-mender-tegra/meta-mender-tegra-jetpack5/recipes-bsp/uefi/edk2-firmware-tegra_%.bbappend

```diff
@@ -0,0 +1,9 @@
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+
+SRC_URI += "file://0007-runtime-access-for-KernelCommandLine-efivar.patch;patchdir=.."
+SRC_URI += "file://0008-enable-rootfs-redundancy.patch;patchdir=.."
```

Comment:
Instead of patch we should use overlays

Reply:
> Instead of patch we should use overlays

Can you provide the change? This is directly based off the `naroj/dev3` branch.

Reply:
Yes, I know and I will.  I marked this line quickly to not forget to do this. Sorry for confusion

Reply:
@maciej-narojczyk if this is working for you the same way it was on your previous branch let's merge and you can propose additional PRs for additional changes.

Reply:
ok